MNT: Re-rendered with conda-build 25.1.2, conda-smithy 3.45.4, and co…

EricEngle-NOAA opened the pull request:
…nda-forge-pinning 2025.02.27.07.39.47

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

EricEngle-NOAA committed:
MNT: Re-rendered with conda-build 25.1.2, conda-smithy 3.45.4, and conda-forge-pinning 2025.02.27.07.39.47

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.
 * ℹ️ The recipe is not parsable by parser `conda-recipe-manager`. The recipe can only be automatically migrated to the new v1 format if it is parseable by conda-recipe-manager.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/13574146514](https://github.com/conda-forge/conda-forge-webservices/actions/runs/13574146514). Examine the logs at this URL for more detail.</sub>

EricEngle-NOAA committed:
Apply patch for g2cio.c

EricEngle-NOAA committed:
Apple patch for g2ccsv.c.

EricEngle-NOAA committed:
Bad patch for g2ccsv.c last time.  New one.

EricEngle-NOAA committed:
Fix for g2ccsv-max-line-size.patch.

EricEngle-NOAA committed:
Fix another type in g2ccsv.c patch file.

EricEngle-NOAA committed:
Another patch, now for g2cindex.c

g2cindex.c uses basename() from libgen.h which only exists for
POSIX.  So here we recreate the basename() function for windows.

EricEngle-NOAA committed:
Fix typo in latest patch.

EricEngle-NOAA committed:
Temporarily stopping Windows builds.

NCEPLIBS-g2c v2.0.0 introduced code in several areas that are not
compatible with Windows (i.e. non-POSIX systems).  Therefore, until
these are "fixed" in the NCEPLIBS-g2c library, we will stop building
for Windows.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ Selectors are suggested to take a ``<two spaces>#<one space>[<expression>]`` form. See lines [14]

For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.
 * ℹ️ The recipe is not parsable by parser `conda-recipe-manager`. The recipe can only be automatically migrated to the new v1 format if it is parseable by conda-recipe-manager.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/13575328549](https://github.com/conda-forge/conda-forge-webservices/actions/runs/13575328549). Examine the logs at this URL for more detail.</sub>

EricEngle-NOAA committed:
MNT: Re-rendered with conda-build 25.1.2, conda-smithy 3.45.4, and conda-forge-pinning 2025.02.27.15.37.26

Forgot to rerender after stopping Windows builds.

EricEngle-NOAA committed:
Remove lint.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.
 * ℹ️ The recipe is not parsable by parser `conda-recipe-manager`. The recipe can only be automatically migrated to the new v1 format if it is parseable by conda-recipe-manager.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/13575525606](https://github.com/conda-forge/conda-forge-webservices/actions/runs/13575525606). Examine the logs at this URL for more detail.</sub>

EricEngle-NOAA commented:
OK...For now, removing Windows builds until NCEPLIBS-g2c can better support non-POSIX platforms.
